index.html

```diff
@@ -0,0 +1,186 @@
+        </div>
+
+        <div class="lecture lecture2">
+            <img class="program-icons" src="Images/karaokes.png" alt="pic">
+            <div class="title"> karaokes</div>
+            <div class="activity">Listen to out speaks from various countries about the message of thinking and the world you know we must carry out all the necessary obligations but if you are not really go there</div>
+        </div>
+
+        <div class="lecture lecture3">
+            <img class="program-icons" src="Images/firework.png" alt="pic">
+            <div class="title">Fireworks</div>
+            <div class="activity">Listen to ouies about the message of thinking and sharing allover the world ust carry out all the necessary obligations but if you are not really go there</div>
+        </div>
+
+        <div class="lecture lecture4">
+            <img class="program-icons" src="Images/stunts.png" alt="pic">
+            <div class="title">Stunting</div>
+            <div class="activity">various countries about the message of thinking and sharing alloverw we must carry out all the necessary obligations but if you are not really go there</div>
+        </div>
+
+        <div class="lecture lecture5">
+            <img class="program-icons" src="Images/clubing.png" alt="pic">
+            <div class="title">Clubing</div>
+            <div class="activity"> message of thinking and sharingm various countries about theorld you know y obligations but if you are not really go there</div>
+        </div> 
+    </div>
+
+    <div class="link">
+       <a  href="#"><div id="aa">SEE THE WHOLE CONCERT</div></a> 
+    </div>
+
+        <button type="button" id="btn">
+            Join BB Big Bang 2021
+        </button>
+    </section>
+
+   <section class="speakers">
+        <h2>Featured Performers</h2>
+        <div class="line-container2">
+        <p class="red-line2">.</p>
+    </div>
+
+    <div class="speakers-container" id="percontainer"> 
+</div>
+
+
+<div class="partners">
+  <h1 class="partner">Partners</h1>
+
+     <div class="lining">
+        <div class="line">.</div>
+    </div>
+
+    <div class="row">
+    <p class="partner1">Mozilla</p>
+    <p class="partner2">Google</p>
+    <p class="partner3">NAVER</p>
+    <p class="partner4">duamkakao</p>
+    <p class="partner5">
+          airbnb
+      </p>
+    </div>
+</div>
+   
+<div class="logo-reservedright">
+    <div class="footlogo">
+        <img id="footlogo-img" src="Images/the-logo.png" alt="pic">
+    </div>
+    <div class="right">
+       <p class="pa"> 2021 Big Bang NewYork. Some Right Reserved</p>
+        <p class="pa"> Our Rights are always reserved at any moments, dont knowthis photograph in this life when yot with time to make show they all copnd fandwith wh at they are saying ihuh ahk ADHCAK OHAKdv sfvflvl iflvld</p>
+    </div>
+</div>
+         <div class="more">
+            MORE <i class="fa-solid fa-angle-down" id="down"></i>
+        </div> 
+  
+    </section> 
```

Comment:
A few changes are needed in this section so that your project organization improves in terms of semantics. First of all, please be sure that you use `footer` tag rather than a `div` tag. Another important change to implement is the use of an `article` tag for each of your dynamic created performers 🧐 This will improve your semantics greatly! Kindly make this changes in your home and about page

Reply:
DONE